Makefile

```diff
@@ -153,7 +153,7 @@ test-heavy:
 
 .PHONY: test-ui
 test-ui:
-	$(MAKE) test-with-database TIMEOUT_M=25 PROVE_ARGS="$$HARNESS t/ui/*.t" GLOBIGNORE="t/*tidy*:t/*compile*:$(unstables)"
+	$(MAKE) test-with-database RETRIES=50 STABILITY_TEST=1 TIMEOUT_M=25 PROVE_ARGS="$$HARNESS t/ui/27-plugin_obs_rsync_status_details.t"
```

Comment:
oops, that should have been `RETRY=50`, sorry about that :/

Reply:
fixed in https://github.com/os-autoinst/openQA/pull/3888/commits/137eec91b12b7de25318f5afdcd2ca94c6589bba